pebblo/app/pebblo-ui/src/components/snippetDetails.js

```diff
@@ -106,17 +106,13 @@ export function SnippetDetails(props) {
   function onChange(evt) {
     let filteredData;
     if (evt.target.value) {
-      filteredData = snippetList?.filter((item) =>
-        eval(
-          searchField
-            ?.map((sch) =>
-              item[sch]
-                ?.toLocaleLowerCase()
-                ?.includes(evt.target.value.toLocaleLowerCase())
-            )
-            .join(" || ")
-        )
-      );
+      filteredData = snippetList?.filter((item) => {
+        const searchValue = evt.target.value.toLocaleLowerCase();
```

Comment:
nitpick: this line can be moved outside the filter to avoid calling toLocaleLowerCase() repeatedly.

Reply:
resolved